apps/site/assets/ts/components/__tests__/__snapshots__/ModalTest.tsx.snap

```diff
@@ -20,7 +20,9 @@ exports[`it renders 1`] = `
         containerInfo={
           <body>
             <div
+              aria-hidden="true"
```

Comment:
In this test, this makes sense because it's rendering a snap after being clicked/opened via enzyme.